### PR TITLE
DHFPROD-7578: Stop a running step/flow from Hub Central

### PR DESCRIPTION
**Pull request by lucianaabc-ml**
DHFPROD-7578: Stop a running step/flow from Hub Central

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

**Review by MSBrignoni — CHANGES_REQUESTED** [review]
it is going well @lucianaabc-ml:
- To make it consistent with the other tooltip ("Run Flow"), could we show the tooltip at the top?

image

**Review by briantang — CHANGES_REQUESTED** [review]
Screen Shot 2022-04-25 at 8 33 22 AM

It looks like if I run the flow and close the modal, the flow will finish but the "Stop" button is still on the panel.

I also don't see this part of the AC. Was there an issue there?

"When I click to stop a flow,
Then the remaining steps in the flow appear and show "Canceled" with explore links disabled."

File: marklogic-data-hub-central/ui/src/components/flows/flows.tsx [hunk 1]
```diff
@@ -891,30 +914,29 @@ const Flows: React.FC<Props> = ({
         formData.append("files", file);
       });
 
-      if (singleIngest) {
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/components/flows/flows.tsx:894]: Looks good now but appears like one of my bugfixes were undone with some conflict resolution here. This previous code is needed to fix the issue with running a single load step.

**Review by briantang — CHANGES_REQUESTED** [review]

**Comment by briantang** [comment]
@lucianaabc-ml Please also update the Wiki with this new functionality: https://wiki.marklogic.com/display/ENGINEERING/Data+Hub+5.7.2

File: marklogic-data-hub-central/ui/src/config/tooltips.config.tsx [hunk 2]
```diff
@@ -484,7 +484,9 @@ const RunToolTips = {
   runEmptyFlow: 'There are no steps in the flow. You must add steps to the flow.',
   flowEdit: 'Edit Flow',
   flowDetails: 'Flow Details',
-  moveLeft: 'Move left'
+  moveLeft: 'Move left',
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/config/tooltips.config.tsx:487]: Minor thing here but the tooltips here look to be the outdated ones. Please refer to the right column of that section in Matthew's UI Text wiki here: https://wiki.marklogic.com/display/ENGINEERING/Data+Hub+5.7.1+UI+Content#DataHub5.7.1UIContent-RunFlows

**Review by briantang — CHANGES_REQUESTED** [review]
We can get the modal to re-render the remaining steps to also show "Canceled" if you add formatExtraData below the formatter. Check below:

formatter: ...,
 formatExtraData: {jobResponse}

**Review by briantang — APPROVED** [review]

**Review by MSBrignoni — APPROVED** [review]

**Pull request merged**